modules/tinymce/src/core/test/ts/browser/keyboard/PageUpDownKeysTest.ts

```diff
@@ -0,0 +1,88 @@
+import { Keys } from '@ephox/agar';
+import { context, describe, it } from '@ephox/bedrock-client';
+import { TinyAssertions, TinyContentActions, TinyHooks, TinySelections } from '@ephox/mcagar';
+import { PlatformDetection } from '@ephox/sand';
+
+import Editor from 'tinymce/core/api/Editor';
+import Theme from 'tinymce/themes/silver/Theme';
+
+describe('browser.tinymce.core.keyboard.PageUpDownKeyTest', () => {
```

Comment:
Given that this logic relies on actual browsers doing something and then us overriding that logic, we really shouldn't be using a simulated test here. This instead should be a webdriver test so that actual keyevents are fired and operated by the browser.

I'd suggest asking for help on how to do that, but the gist is that you'll need to:
- move this to the `webdriver` directory
- update the `describe` to say `webdriver` instead of `browser`
- use `RealKeys` from agar instead of using `TinyContentActions` to make webdriver fire real key events